docker/tracker_dcs_stack/docker-compose.dummy.yml

```diff
@@ -0,0 +1,47 @@
+version: "3"
+services:
+  hv:
+    image: ghcr.io/cdozen/dummy:0.0.1
```

Comment:
use the official repo in our organization

Reply:
I understand why you used yours, but please change this after we merge all PRs

Reply:
I tried to use offical repo but it gives access denied error

Reply:
to be solved, then